apps/examples/testcase/le_tc/network/tc_net_pkt.c

```diff
@@ -0,0 +1,160 @@
+/*  @file tc_net_socket.c
+ * @brief Test Case Example for pbuf() API
+ */
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <errno.h>
+
+#include <sys/stat.h>
+#include <net/if.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <netutils/netlib.h>
+
+#include <sys/socket.h>
+#include <lwip/pbuf.h>
+
+#include "tc_internal.h"
+
+#define PBUF_TEST 6
+
+/**
+* @testcase             tc_net_pkt_layer_transport_p
+* @brief		check for the memory allocation for the pbuf by transport layer
+* @scenario
+* @apicovered           pbuf_alloc()
+* @precondition
+* @postcondition
+*/
+void tc_net_pkt_layer_transport_p(void) {
+
+	struct pbuf *p = NULL;
+	p = pbuf_alloc(PBUF_TRANSPORT, PBUF_TRANSPORT_HLEN, PBUF_RAM);
+	TC_ASSERT_NEQ_CLEANUP("pbuf", p, NULL, pbuf_free(p));
+	TC_SUCCESS_RESULT();
+
+	pbuf_free(p);
+}
+
+/**
+* @testcase             tc_net_pkt_layer_ip_p
+* @brief		check for the memory allocation for the pbuf by ip layer
+* @scenario
+* @apicovered           pbuf_alloc()
+* @precondition
+* @postcondition
+*/
+void tc_net_pkt_layer_ip_p(void) {
+
+	struct pbuf *p = NULL;
+	p = pbuf_alloc(PBUF_IP, PBUF_IP_HLEN, PBUF_RAM);
+	TC_ASSERT_NEQ_CLEANUP("pbuf", p, NULL, pbuf_free(p));
+	TC_SUCCESS_RESULT();
+
+	pbuf_free(p);
+}
+
+/**
+* @testcase             tc_net_pkt_layer_link_p
+* @brief		check for the memory allocation for the pbuf by link layer
+* @scenario
+* @apicovered           pbuf_alloc()
+* @precondition
+* @postcondition
+*/
+void tc_net_pkt_layer_link_p(void) {
```

Comment:
Please find our coding rules. You should do as below.
```
void xxx(void)
{
```

Reply:
done